ddtrace/profiling/collector/memalloc.py

```diff
@@ -49,7 +49,7 @@ class MemoryHeapSampleEvent(event.StackBasedEvent):
 
 
 def _get_default_heap_sample_size(
-    default_heap_sample_size=512 * 1024,  # type: int
+    default_heap_sample_size=1024 * 1024,  # type: int
```

Comment:
The changes in this source seem unrelated to the PR?

Reply:
No, it's related. We want to only store a sample every 1 MB of allocated memory rather than 512 KB.